fe/fe-core/src/test/java/com/starrocks/clone/DiskUsageSimTest.java

```diff
@@ -0,0 +1,93 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present StarRocks Inc.
+package com.starrocks.clone;
+
+import com.starrocks.common.Config;
+import com.starrocks.pseudocluster.PseudoBackend;
+import com.starrocks.pseudocluster.PseudoCluster;
+import com.starrocks.pseudocluster.PseudoClusterUtils;
+import com.starrocks.server.GlobalStateMgr;
+import com.starrocks.system.SystemInfoService;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.sql.SQLException;
+
+public class DiskUsageSimTest {
+    private final long GB = (1L << 30);
+    @BeforeClass
+    public static void setUp() throws Exception {
+        // set some parameters to speedup test
+        Config.tablet_sched_checker_interval_seconds = 1;
+        Config.tablet_sched_repair_delay_factor_second = 1;
+        Config.enable_new_publish_mechanism = true;
+        PseudoCluster.getOrCreateWithRandomPort(true, 3);
+        GlobalStateMgr.getCurrentState().getTabletChecker().setInterval(1000);
+        PseudoCluster cluster = PseudoCluster.getInstance();
+        cluster.runSql(null, "create database test");
+        cluster.runSql("test",
+                "create table test ( pk bigint NOT NULL, v0 string not null, v1 int not null )" +
+                        " primary KEY (pk) DISTRIBUTED BY HASH(pk) BUCKETS 1 " +
+                        "PROPERTIES(\"replication_num\" = \"3\", \"storage_medium\" = \"SSD\");");
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        PseudoCluster.getInstance().shutdown(true);
+    }
+
+
+    @Test
+    public void testSetInitialDiskCapacity() throws InterruptedException {
+        PseudoCluster cluster = PseudoCluster.getInstance();
+        SystemInfoService systemInfoService = GlobalStateMgr.getCurrentSystemInfo();
+        // test default value
+        Assert.assertEquals(PseudoBackend.DEFAULT_TOTA_CAP_B,
+                systemInfoService.getBackend(10001).getTotalCapacityB());
+        PseudoBackend backend1 = cluster.getBackend(10001);
+        backend1.setInitialCapacity(10 * GB, 8 * GB, 2 * GB);
+        PseudoBackend backend2 = cluster.getBackend(10002);
+        backend2.setInitialCapacity(20 * GB, 8 * GB, 6 * GB);
+        // wait for the disk info to be reported
+        Thread.sleep(10000);
```

Comment:
My following patch will make report time configable, and so there may be some mechanism to wait disk info to be reported rather than a fixed sleep(which is not reliable)

Reply:
Not requesting a change here, maybe add a TODO

Reply:
ok, i will mark a todo here